Lecture5/Task1/Helper/Operations.java

```diff
@@ -0,0 +1,23 @@
+package Helper;
+
+public class Operations {
+
+    private Operations() {
+    }
+
+    public static double getSum(double firstNumber, double secondNumber) {
+        return firstNumber + secondNumber;
+    }
+
+    public static double getDiff(double firstNumber, double secondNumber) {
```

Comment:
why diff and not substraction

Reply:
fixed
Changed a method name to getDifference.
Because subtraction is an arithmetic operation that represents the operation of removing objects from a collection. The result of a subtraction is called a difference. And method returns deffirence of two numbers